packages/app/src/systems/CRX/background/services/BackgroundService.ts

```diff
@@ -93,7 +93,7 @@ export class BackgroundService {
     } else {
       await this.requireAccounts();
     }
-    return next(request, serverParams);
+    return next(request, serverParams!);
```

Comment:
You can change the argument here: `serverParams: EventOrigin | undefined`. to `serverParams: EventOrigin`